spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -90,7 +90,9 @@
         "spark",
         SparkInterpreter.class.getName(),
         new InterpreterPropertyBuilder()
-            .add("spark.app.name", "Zeppelin", "The name of spark application.")
+            .add("spark.app.name",
```

Comment:
Thinking about it just a *bit* more, it would be nice to have the env setting be a way of changing the default, but still allow the user to override it themselves.  If I set it in interpreters.json (as well as spark master that way), I'll have to keep track of the first time I run vs. subsequent runs so I don't interfere with user settings.  I can imagine that users may want to, eventually, start their own spark instance/cluster to run against and override that setting.

Cheers,
Craig


> On Nov 8, 2015, at 16:29, Craig Ching <craigching@gmail.com> wrote:
> 
> Hi Felix,
> 
> But I also set MASTER (spark master) that way too.  It seems you guys have a lot of different ways to set things and I was just trying to do what was accepted.  The way I start zeppelin (I run zeppelin in a docker container), it's much easier to create the env file on the fly than to have to parse a large-ish json file.  But whatever, I asked on the user list and moon thought it might be a good idea, I just want to be consistent.  This seems like an area that you could tighten up, which is the correct way to specify these settings, zeppelin-env.sh, zeppelin-site.xml, or interpreter.json.  Certainly interpreter.json is not the documented way to make these settings, at least it wasn't obvious to me.
> 
> But whatever, I'll retract the PR then.
> 
> Cheers,
> Craig
> 
> Sent from my iPad
> 
>> On Nov 8, 2015, at 16:14, Felix Cheung <notifications@github.com> wrote:
>> 
>> In spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java:
>> 
>> > @@ -90,7 +90,9 @@
>> >          "spark",
>> >          SparkInterpreter.class.getName(),
>> >          new InterpreterPropertyBuilder()
>> > -            .add("spark.app.name", "Zeppelin", "The name of spark application.")
>> > +            .add("spark.app.name",
>> Sure. If you are to set SPARK_APP_NAME in zeppelin-env.sh, you could also set spark.app.name in interpreter.json also in /conf?
>> 
>> —
>> Reply to this email directly or view it on GitHub.
>>